### PR TITLE
Image added

### PR DESCRIPTION
**Pull request by Swapnilkota108**

### CODE REVIEW

File: components/Images/images.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    width: 50%;
+}
+
+
+.img{
+   width: 17rem;
+   height: 17rem;
+   object-fit: cover;
+}
+  .img-rd{
```
Comment (mimansha-swarup) [inline components/Images/images.css]: kee more meaningful name like round instead of rnd,responsive instead of rsv

**Review by mimansha-swarup — COMMENTED** [review]
few suggestions

File: components/Images/images.css (hunk 1 above)
Comment (Swapnilkota108) [reply components/Images/images.css]: Sure. Thanks for the suggestion.

**Review by srishti-maurya — APPROVED** [review]
You can add a preview link may be in the PR

**Pull request merged**